### DISCUSSION TITLE
Ошибка при экспорте .ogf.

### DISCUSSION THREAD

**Discussion by SamArtGit** [PavelBlend/blender-xray#441] [Q&A] [answered]
Открыл оригинальную модель .ogf из ТЧ и пересадил лицо из модели со слива объектов от Лохотрона .object ТЧ.
Самая последняя версия аддона и Блендера (Steam).

![image](https://user-images.githubusercontent.com/56885075/139540058-b52fdb8a-8ee8-49c4-83d2-896abb7a1a02.png)

[Bug.zip](https://github.com/PavelBlend/blender-xray/files/7447038/Bug.zip)

**Comment by SamArtGit** [comment]
Разобрался. Не была назначена арматура для головы. Интересно, возможно ли сделать расшифровку для таких типов ошибок?

**Comment by PavelBlend** [comment]
@SamArtGit позже буду пересматривать экспорт ogf. Попытаюсь сделать так, чтобы экспортёр нормально обрабатывал подобные случаи.

Но пока лучше использовать object экспорт. Так как экспортёр в ogf не сохранит swis. А СДК сохранит.

**Comment by PavelBlend** [comment, marked as answer]
Исправил эту ошибку. Фикс будет доступен в следующем релизе 2.6.0. Аддон теперь будет выдавать такую читаемую ошибку:
![01](https://user-images.githubusercontent.com/7983249/144022596-da3599e7-6968-4981-ae89-1b9f5c9a43ae.jpg)

И как обычно, более подробно ошибки описываются в текстовом редакторе блендера:
![02](https://user-images.githubusercontent.com/7983249/144022664-f19df75d-6784-480b-9d88-5beadf7d58ac.jpg)
